### DISCUSSION TITLE
PC起動時キーボードを認識しない

### DISCUSSION THREAD

**Discussion by YOUYOU41** [Yowkees/keyball#374] [Q&A] [answered]
２回目の質問失礼します
PCを起動したときに毎回USBの抜き差しかリセットボタンを２回押さないとキーボードが認識しません
そこで別の方の書き込みでありました

config.h内の下記の個所をコメントアウトしたところ無事にPC起動時何もしなくても認識するようになったのですが
//#define SPLIT_USB_DETECT
//#define SPLIT_USB_TIMEOUT      500

これだとremapが使えません
remapを使いたいのですが何か方法はないのでしょうか？
今までプログラム経験がなくいろいろ調べてやっとここまではたどり着くことが出来たのですが
これ以上はどうしてもわかりません
お手数ですがよろしくお願いいたします。

**Comment by Yowkees** [comment, marked as answer]
私も上記のソースコード変更を試したことがなく、適切な回答ができるかわかりませんが、
ファームウェアをビルドするときのキーボード名を
keyball/keyball61:via
としてもREMAPが使用できないでしょうか？

お手数ですがご確認お願いいたします。

**Comment by YOUYOU41** [comment]
ご回答ありがとうございました

keyball/keyball61:viaでビルドしたところ普通にREMAPを使用することが出来、
PC起動時も何もせずにキーボドが起動すようになりました。
いままでkeyball/keyball61:defaultでmakeしていたためremapが使用出来なかったみたいです

これですべての問題が解決したので安心してkeyball生活がおくれそうです
本当にありがとうございました

**Comment by Yowkees** [comment]
無事にREMAPが使用できたようで安心しました。
ぜひ末永くご愛用いただければ幸いです。
